opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/AccountInformationRequestCommon.java

```diff
@@ -165,6 +166,14 @@ public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_wi
         return self();
     }
 
+    public SELF user_anton_brueckner_provided_initial_parameters_to_list_accounts_with_all_accounts_consent_without_psu_id(Approach expectedApproach) {
```

Comment:
Method name clearly says that no `psu id` is going to be used - you should provide method and separate test that does not send PSU id